app/admin/concepts/statistic/operation.rb

```diff
@@ -0,0 +1,45 @@
+class StatisticOperation
+  class EventCount
+    class Show
+      def call(params = {})
+        Event.count
+      end
+    end
+  end
+
+  class TotalParticipants
```

Comment:
Oh, good point about the `volunteers_count` @pferdefleisch, did not think about this. Totally fine then to not be more explicit about I'd say. But as you like, maybe it's a useful reminder in some months then.